### DISCUSSION TITLE
我想问一下绳网应用登录时显示404怎么解决？

### DISCUSSION THREAD

**Discussion by minecraft4668** [share121/inter-knot#4770] [科技]
![0ab04bcfd7308cb7de2a2c2ab43b3a79](https://github.com/user-attachments/assets/c0a6dce0-e830-4d75-a8d3-3abe7fd6cc94)
（随便放张图做标题图片吧~）
![image](https://github.com/user-attachments/assets/90c789e3-ed0e-4a0e-b332-56332f262a5b)
想用电脑端的绳网的时候遇到了这个怎么解决？可以有大佬来解决吗？

**Comment by tuneinnocent** [comment]
手机端也是如此，点评论进入网页才能看到里面的图。

**Comment by Hangover-Cyan-Eternally** [comment]
今天dy上刷到了这个，搞了半天终于登录成功了

**Reply by minecraft4668** [reply]
佬怎么做到的，教我IwI

**Comment by minecraft4668** [comment]
这个客户端好像会被识别成**病毒**（

**Comment by Hangover-Cyan-Eternally** [comment]
不是，我是菜鸡，昨天是搞了个GitHub注册了好久😭

**Reply by HKLHaoBin** [reply]
没关系，至少你愿意折腾

**Reply by HLXS-HLXS** [reply]
你能注册成功就已经比绝大多数人强了，别妄自菲薄<img src="https://bbs-static.miyoushe.com/static/2023/12/22/2a010db5d2ab94345ef4f69bad152122_3253943580882084522.png" alt="月下-爱你" width="75" >

**Comment by minecraft4668** [comment]
我在想我是不是挂错了，应该挂到委托类的......

**Comment by minecraft4668** [comment]
e好吧，最新版本已经解决了
